portfolio/src/main/java/com/google/sps/servlets/RandomStringServlet.java

```diff
@@ -1,20 +1,38 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
-import java.util.Date;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that responds with the current date. */
+/** Servlet that shows a random string */
 @WebServlet("/string")
 public class RandomStringServlet extends HttpServlet {
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Random string 1");
-    response.getWriter().println("Random string 2");
+    String string1 = "My favorite movie is the notebook";
+    String string2 = "I like chocolate";
+    String string3 = "My favorite song is mirrors by justin timberlake";
+
+    //String str = [string1, string2, string3];
+    //String string = new String(String1, String2, String3);
+     String json = convertToJson(string1);
+     String json1 = convertToJson(string2);
+     String json2 = convertToJson(string3);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
```

Comment:
My instructions from yesterday were trying to move this off to a separate review, but it looks like it got submitted. I wanted to provide some feedback anyways.

Generally when returning json from the server, it should be a single json document rather than multiple json documents. That way the client can just decode the response once and then work with that data, rather than trying to figure out how many different items need to be decoded independently.

I realize that without using a json library, it is probably hard to do this, so what you have here seems fine for now. In the future, it will probably simplify things to make use of a library which transforms your objects into json. That way, you can just pass it the right data structure to encode all the data at once.

Reply:
Even without a library, you could probably hand-code a single json document which includes your 3 strings though.